national-resources/sweden.md

```diff
@@ -0,0 +1,121 @@
+- **[The National Board for Health and  Welfare (Socialstyrelsen)](https://www.socialstyrelsen.se/en/)** is responsible for issuing guidelines and regulations related to public health. It also maintains multiple [clinical registers](https://www.socialstyrelsen.se/en/statistics-and-data/registers/) and [statistics on COVID-19](https://www.socialstyrelsen.se/en/statistics-and-data/statistics/statistics-on-covid-19/).
+
+## Disease surveillance initiatives
+
+Multiple initiatives have been established for disease surveillance in Sweden. Below we give examples of ongoing initiatives. These initiatives generally monitor a broad range of pathogens.
+
+### Wastewater surveillance
+
+Wastewater surveillance has proven to be an effective means of predicting outbreaks, with peaks in viral genome content shown to precede increases in cases and hopitalisations. This is also true of Swedish wastewater surveillance efforts, with  e.g. [Wang *et al.* (2022)](https://pubmed.ncbi.nlm.nih.gov/36035197/) showing that peaks in viral genome content preceded increases in cases by two weeks.
+
+Multiple research groups across Sweden have monitored viral genome content in wastewater, including groups at the [University of Gothenburg](https://www.gu.se/en), [KTH Royal Institute of Technology](https://www.kth.se/en), and [Swedish University of Agricultural Sciences (SLU)](https://www.slu.se/en/). Throughout the COVID-19 pandemic these efforts largely focused on monitoring SARS-CoV-2, but the scope has expanded to include other pathogens. More information on the work by these groups is on the [Swedish COVID-19 and Pandemic Preparedness Portal](https://www.covid19dataportal.se/dashboards/wastewater/).
+
+### Sentinel monitoring in healthcare
+The Swedish Public Health Agency coordinates [sentinel monitoring (information only available in Swedish)](https://www.folkhalsomyndigheten.se/mikrobiologi-laboratorieanalyser/mikrobiella-och-immunologiska-overvakningsprogram/sentinelovervakning/) for COVID-19 and influenza. In sentinel monitoring, samples are taken from patients with symptoms of influenza or acute respiratory infections. These samples are analysed by [The Swedish Public Health Agency](https://www.folkhalsomyndigheten.se/the-public-health-agency-of-sweden/). The data is then used to inform national policy decisions, and is reported to the public, before being subsequently reported to the [European Centre for Disease Prevention and Control (ECDC)](https://www.ecdc.europa.eu/en). ECDC then reports the date to the [World Health Organisation (WHO)](https://www.who.int/). 
+
+## Dashboards and visualisation platforms
+
```

Comment:
We debated this at the contentathon too, the decision across the national pages was to add individual links to each dashboard, so that's why we've left it like this. However, I am happy to change it to be more concise. Perhaps this is a rule that we need to come to at an editor level? Like 'if there are multiple pages all related to the same website then they should be presented as '...' ?